easybuild/easyconfigs/t/tabix/tabix-0.2.6-foss-2019b.eb

```diff
@@ -0,0 +1,37 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
```

Comment:
Please restore the authorship of previous easyconfigs of tabix. You can add your authorship to the list of authors found in the last one, which is `tabix-0.2.6-GCCcore-7.3.0.eb`.

Reply:
Done. Thanks for spotting this, this was not how I intend to do it.